main.js

```diff
@@ -0,0 +1,26 @@
+    let els = document.querySelectorAll("header nav ul li");
+    for(let el of els){
+        if(el.textContent.includes("All Posts")){
+            el.remove();
+        }
+    }
```

Comment:
```suggestion
document.querySelector("li").remove();
```
Since the "All Posts" link happens to be the first `<li>`, you can just write it like this. However, I do like that you were very thorough and used a for loop/if statement. This way the "All Posts" link will be removed whether it's the first `<li>` or not.

Reply:
@samshapland22 
>
 I really appreciate you taking time out of you day to share some constructive / positive comments!
 
Whilst solving this i wanted to make it as dynamic as possible so that id grab multiple elements instead of just the first one although there was only one. I thought "what if I don't know theres only one element" and it clicked.